aten/src/ATen/native_parse.py

```diff
@@ -129,6 +129,10 @@ def type_argument_translations(arg):
         default = 'Reduction::Mean'
     elif default == 'contiguous_format':
         default = 'MemoryFormat::Contiguous'
+    elif default == 'NoQuant':
+        default = 'QScheme::NO_QUANT'
+    elif default == 'PerTensorAffine':
```

Comment:
this should match the python specification as closely as possible, i.e. `per_tensor_affine`.